src/puppetlabs/puppetdb/http/metrics/core.clj

```diff
@@ -1,4 +1,4 @@
-(ns puppetlabs.puppetdb.http.metrics
+(ns puppetlabs.puppetdb.http.metrics.core
```

Comment:
I'm almost tempted to say drop the http from the namespace here, and just make it puppetlabs.puppetdb.metrics.* for everything, then put say your service (which is the entry point really) in puppetlabs/puppetdb/metrics.clj ... gives it sort of a nice hierarchy, and since the http namespace has PDB specific stuff in it, kind of separates it from the rest of the query stuff.